examples/search/public/client.js

```diff
@@ -0,0 +1,17 @@
+const search = document.querySelector('#search')
+const code = document.querySelector('pre')
+
+search.addEventListener(
```

Comment:
please use the newer `fetch` API for the example.

`XMLHttpRequest` is kind of old